zerver/forms.py

```diff
@@ -242,28 +201,60 @@ def save(self,
         email in multiple subdomains, we may be able to delete or refactor this
         function.
 
+        We inject request parameter in context. This parameter will be used by
+        ZulipPasswordResetForm.send_mail and
+        ZulipPasswordResetForm.send_mail_to_non_existent_user function.
```

Comment:
Are we using `send_mail_to_non_existent_user`?

Reply:
Nope :)